tests/common.h

```diff
@@ -60,10 +60,22 @@ static inline void test_time_start(struct timespec* start) { clock_gettime(CLOCK
 void test_time_end(struct timespec* start, struct timespec* end, double* sum);
 
 /**
- * Generate a random trytes combination with given length
+ * @brief Initialize random number generator. Call it before using 'rand_trytes_init()'.
```

Comment:
Call it before using `rand_trytes_init()`?

Reply:
I mean isn't it supposed be `Call it before using gen_rand_trytes()`?

Reply:
But the first sentence describes what's the functionality of this function

Reply:
You describe the function `rand_trytes_init()` that it should be called `before using itself`?
That does not make sense.

Reply:
oh my fault